src/test/kotlin/mediathek/config/StandardLocationsTest.kt

```diff
@@ -22,14 +24,24 @@ internal class StandardLocationsTest {
         assertTrue { testPath == xmlFilePath }
     }
 
+    @Test
+    fun getXDGDownloadDirectory() {
```

Comment:
this test should also check expected results on windows and macOS and not just only Linux. Otherwise you might miss (future) unexpected behaviour on those platforms.

Reply:
Good point. I'll update this test and also look at the pipeline failure.

Reply:
Fixed.